backend/likes/schemas.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel
+
+
+class LikeCreate(BaseModel):
+    like: bool = None
```

Comment:
@yus252 
this boolean represent like when it's true,
dislike when false,
and nothing when None

Reply:
How do we get the counts of like and dislike for each mystory?